include/algorithms/decision_tree/decision_tree_regression_training_types.h

```diff
@@ -75,6 +75,7 @@ enum InputId
 {
     data                            = algorithms::regression::training::data,               /*!< %Input data table */
     dependentVariables              = algorithms::regression::training::dependentVariables, /*!< Values of the dependent variable for the input data */
+    weights                         = algorithms::regression::training::weights,            /*!< Optional. Weights of the observations in the training data set */
```

Comment:
This change possible broke dynamic backward compatibility.

Reply:
fixed